### DISCUSSION TITLE
I can t download LC09_L1GT_133043_20230623_20230623_02_T2 error

### DISCUSSION THREAD

**Discussion by mewcamo** [community/community#58961] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

This erore but I can download year 2022 Now I Can t use  landsatxplore download year 2023 

Download failed with dataset id 1 of 2. Re-trying with the next one.
None of the archived ids succeeded! Update necessary!
LC09_L1GT_133043_20230623_20230623_02_T2 error

import os
from landsatxplore.earthexplorer import EarthExplorer 
from landsatxplore.api import API
username = "XXXXX"
password = "XXXX5"
print(username ,password)
api = API(username, password)
response = api.request(endpoint="dataset-catalogs")
print(response)
scenes = api.search(
    dataset='landsat_ot_c2_l1',
    latitude=24.556,
    longitude=96.9,
    start_date='2023-06-22',
    end_date='2023-06-30',
    max_cloud_cover=100
)
df_scenes = pd.DataFrame(scenes)
df_scenes = df_scenes[['display_id','wrs_path', 'wrs_row','satellite','cloud_cover','acquisition_date']]
df_scenes.sort_values('acquisition_date', ascending=False, inplace=True)
ee = EarthExplorer(username, password)
ID_2 = str(df_scenes.display_id)
ID=ID_2[5:45]
ID

try: 
    ee.download(ID, output_dir='E:/')
    print('{} succesful'.format(ID))
except:
    if os.path.isfile('{}.tar'.format(ID)):
        print('{} error but file exists'.format(ID))
    else:
        print('{} error'.format(ID))

ee.logout()

**Comment by ozeze12** [comment]
Good job

**Comment by DSDmark** [comment]
Hi @mewcamo,

Did you verify that **Landsat Explorer** is `support` downloading data from 2023 or not?

**Reply by mewcamo** [reply]
I m not sure sire.

**Reply by DSDmark** [reply]
Try to take between `01/01/2020 to 12/31/2022`

**Reply by mewcamo** [reply]
2022 can take sir
